### ISSUE TITLE
Provide torrents

### ISSUE THREAD

**Issue by blobgo** [berzerk0/Probable-Wordlists#4]
Instead of Mega links maybe providing torrents would also be nice. E.g. Mega requires an add-on for downloading files >1GB.

**Comment by iancnorden** [comment]
What would be the best way to handle revisions via torrents that get pushed up here? Just new links and an announce of updates similar to Kali?

**Comment by berzerk0** [comment]
Torrents will go up and I'll just push new versions of them as files change.
Kali seems like a good system, @iancnorden. Why not do the same?

**Comment by iancnorden** [comment]
I was referring to how Kali handled their torrents, kill off seeding any olds, announce their new ones, single source of seed. 

Not much upload to throw at torrents currently, but can help seed your initial. Will be looking to you to create the initial torrent file and seed so there is no confusion on who is putting this out, happy to help provide guidance if you haven't done so before.

**Comment by berzerk0** [comment]
In regards to Kali, that was my understanding, @iancnorden. I think you came up with a good idea.

I'm starting with Dictionary Style Torrents to see how it goes

**Comment by iancnorden** [comment]
Ah thanks, and yea start small to test is a good idea, I can help test make sure up / down works etc before pushing an 'official' one on this repo.

**Comment by berzerk0** [comment]
Seedbox is going up, expect torrents within 5 days

**Comment by berzerk0** [comment]
All files placed on (in?) the box and ready to seed.
I've got to actually include the torrents here on the repo - but that'll be done today for sure.

I will admit that I am new to being the seed master, but I was able to reliably get the torrents at least trickling to a few test buddies. There are plenty of trackers and the seedbox claims to have unlimited bandwidth, so hopefully as the number of peers goes up we'll actually have torrenting be faster than Mega.nz links.

The Mega.nz links will remain, however - as a reliable alternative.
